all-in-one/Dockerfile

```diff
@@ -66,19 +66,13 @@ RUN if [ -n "$REST_PATH" ]; \
     mv oscal-rest-service-app/target/oscal-rest-service-app*.jar "/app/oscal-rest-service-app.jar"; \
     fi
 
-FROM openjdk:${JAVA_VERSION}-slim-bullseye
+FROM gcr.io/distroless/java11-debian11
```

Comment:
Hmm so it looks like based on the [`BUILD`](https://github.com/GoogleContainerTools/distroless/blob/main/java/BUILD) file that this just installs the `openjdk-${ver}-jre-headless` package in the Debian environment. I wonder what impact that has compared to Temurin (fka AdoptOpenJDK) for example that fetches the latest build. Will there be a delay in patches?